tests/libconfigmap/configmap.go

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright The KubeVirt Authors.
+ *
+ */
+package libconfigmap
+
+import (
+	kubev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// New return ConfigMap
+func New(name string, data map[string]string) *kubev1.ConfigMap {
```

Comment:
Although this function is very simple, could you please add a unit test to it?
Something along the line of https://github.com/kiagnose/kubevirt-dpdk-checkup/blob/9ecc55e13b1aa5167ab25d5407650672739e2dc6/pkg/internal/checkup/configmap/configmap_test.go#L35